### DISCUSSION TITLE
Show YAML tags in card (as Linked Page Metadata)

### DISCUSSION THREAD

**Discussion by lguenth** [mgmeyers/obsidian-kanban#588] [Ideas & Feature Requests]
I've recently started to track both my project tasks and my notes' status with this plugin.

I usually track tags in YAML frontmatter  and thought it would be nice to add them to the Linked Page Metadata so I could always see them in my board.

However, tags only seem to appear if they're added inline instead of in the frontmatter.

So my question is: Is it possible to get tags from the frontmatter into my kanban cards? And, if yes, how do I implement it?

(I'm assuming this has something to do with parsing YAML arrays and there are a billion ways to create them, so maybe that's the issue?)

**Comment by mgmeyers** [comment]
Hmm, you're right, there is some weirdness with this. Not sure how much of it is obsidian vs. this plugin. This works, though:

```
---
tags: [my-tag, my-other-tag]
---
```

**Reply by lguenth** [reply]
Ohhhhh. Yes, you're right. Brilliant, thank you! :heart: I feel like I should've tested it more, so sorry for asking here directly. :see_no_evil: 
`tags: ["tag1", "tag2"]` also seems to work but just `tags: my-tag` didn't. That's where I was confused.
